src/main/java/mate/academy/Main.java

```diff
@@ -1,16 +1,45 @@
 package mate.academy;
 
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import mate.academy.lib.Injector;
+import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
+import mate.academy.model.MovieSession;
+import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
+import mate.academy.service.MovieSessionService;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    private static final Injector injector = Injector.getInstance("mate.academy");
+    private static final MovieService movieService =
+            (MovieService) injector.getInstance(MovieService.class);
+    private static final CinemaHallService cinemaHallService =
+            (CinemaHallService) injector.getInstance(CinemaHallService.class);
+    private static final MovieSessionService movieSessionService =
+            (MovieSessionService) injector.getInstance(MovieSessionService.class);
 
+    public static void main(String[] args) {
         Movie fastAndFurious = new Movie("Fast and Furious");
         fastAndFurious.setDescription("An action film about street racing, heists, and spies.");
         movieService.add(fastAndFurious);
         System.out.println(movieService.get(fastAndFurious.getId()));
         movieService.getAll().forEach(System.out::println);
+
+        CinemaHall cinemaHall = new CinemaHall();
+        cinemaHall.setDescription("Hall with blue sits");
+        cinemaHall.setCapacity(100);
+        cinemaHallService.add(cinemaHall);
+
+        MovieSession movieSession = new MovieSession();
+        movieSession.setMovie(fastAndFurious);
+        movieSession.setCinemaHall(cinemaHall);
+        movieSession.setShowTime(LocalDateTime.now());
+        movieSessionService.add(movieSession);
+
+        System.out.println(movieSessionService.get(1L));
+        System.out.println(movieSessionService.findAvailableSessions(1L, LocalDate.now()));
```

Comment:
let's use ids of your created objects instead of 1L - they were already set by Hibernate